js/app/modules/filter/type.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 Endless Mobile, Inc.
+
+/* exported Type */
+
+const Eknc = imports.gi.EosKnowledgeContent;
+const GObject = imports.gi.GObject;
+
+const Filter = imports.app.interfaces.filter;
+const Module = imports.app.interfaces.module;
+
+/**
+ * Class: Type
+ * Filter that includes only a content type
+ *
+ * This filter includes only object models of the content type
+ * specified in its `type` string property.
+ *
+ * The type property accepts a trailing "*" wildcard.
+ *
+ * Using the "invert" property has no effect on this Filter.
```

Comment:
I see - I guess this is the reason why we wanted to have a `not_content_type` filter. In that case, I think it'd be worth implementing that as well for the sake of not breaking interface contract - I am happy to volunteer.

Reply:
Yeah, it should be trivial, let me discuss it in SDK standup (in a some minutes), it might just do it myself if that is the case.